helm/charts/hpe-csi-driver/README.md

```diff
@@ -65,11 +50,7 @@ helm repo update
 
 Install the latest chart:
 ```
-# Helm 3
 helm install hpe-csi hpe/hpe-csi-driver --namespace kube-system -f myvalues.yaml
-
-# Helm 2, for Nimble, Install with HPENodeInfos CRD's enabled explicitly
-helm install --name hpe-csi hpe/hpe-csi-driver --namespace kube-system -f myvalues.yaml --set crd.nodeInfo.create=true
 ```
```

Comment:
Should we leave a note here that says namespace and myvalues.yaml is optional?

Reply:
sure..